### DISCUSSION TITLE
Make existing png image completely transparent

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#3378] [Help] [answered]
Hi all. I just want to clear an existing image of all color so just completely transparent image, but I want to keep the any properties of the image (resolution, color profile, ... and so forth). Any help is appreciated.

**Comment by fmw42** [comment, marked as answer]
On ImageMagick 6,

`convert your_image -alpha transparent your_image
`
On ImageMagick 7, replace convert with magick.

See

https://legacy.imagemagick.org/Usage/masking/#alpha

For novices, see

http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=9620
http://www.imagemagick.org/script/command-line-processing.php
https://imagemagick.org/script/command-line-options.php
http://www.imagemagick.org/Usage/reference.html
http://www.imagemagick.org/Usage/
https://github.com/ImageMagick/usage-markdown
https://imagemagick.org/script/porting.php#cli

**Reply by ghost** [reply]
Yes that worked. I was overthinking this. Thanks so much!
